### PR TITLE
Updating docs to latest

### PR DESCRIPTION
**Pull request by cbmaiolo**

### CODE REVIEW

File: Classes.html [hunk 1]
```diff
@@ -443,337 +455,327 @@ <h4>Declaration</h4>
...
+list of locations that is an efficent path though a venue.</p>
 
-                        <a href="Classes/Category.html" class="slightly-smaller">See more</a>
+<p>This solver cannot just give the best solution, as this a brute force
+solver would run in O(N!) time, which is not resonable if the number
+is bigger then half a dozen items.</p>
+
+<p>Instread of solving the problem all at once, the solver improves
+the solution continiouslly until a good enough solution is found.</p>
+
```
Comment (christiefelker) [inline Classes.html:679]: A few typos here. 'Instread' and 'continiouslly'

File: Classes/PathSolver.html [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+              </div>
+            <p>The PathSolver is used to take a set of locations and create an ordered
+list of locations that is an efficent path though a venue.</p>
+
+<p>This solver cannot just give the best solution, as this a brute force
+solver would run in O(N!) time, which is not resonable if the number
+is bigger then half a dozen items.</p>
+
+<p>Instread of solving the problem all at once, the solver improves
+the solution continiouslly until a good enough solution is found.</p>
```
Comment (christiefelker) [inline Classes/PathSolver.html]: Same comment as above

File: Classes/PathSolver.html [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+                            <tr>
+                              <td>
+                                <code>
+                                <em>origin</em>
+                                </code>
+                              </td>
+                              <td>
+                                <div>
+                                  <p>The place where the solution should start from.
+                 the solver will always make this a loop and circle
```
Comment (christiefelker) [inline Classes/PathSolver.html]: Capitalization

File: docsets/MappedIn.docset/Contents/Resources/Documents/Classes.html [hunk 4]
```diff
@@ -443,337 +455,327 @@ <h4>Declaration</h4>
...
+                        <p>The PathSolver is used to take a set of locations and create an ordered
+list of locations that is an efficent path though a venue.</p>
 
-                        <a href="Classes/Category.html" class="slightly-smaller">See more</a>
+<p>This solver cannot just give the best solution, as this a brute force
+solver would run in O(N!) time, which is not resonable if the number
+is bigger then half a dozen items.</p>
+
+<p>Instread of solving the problem all at once, the solver improves
+the solution continiouslly until a good enough solution is found.</p>
```
Comment (christiefelker) [inline docsets/MappedIn.docset/Contents/Resources/Documents/Classes.html]: Same comment as above

**Review by christiefelker — CHANGES_REQUESTED** [review]

File: Classes.html (hunk 1 above)
Comment (cbmaiolo) [reply Classes.html:679]: good catch

**Review by christiefelker — APPROVED** [review]

**Pull request merged**